authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/AuthorizationConfig.java

```diff
@@ -0,0 +1,101 @@
+import org.apache.gravitino.config.ConfigEntry;
+
+public class AuthorizationConfig extends Config {
+  public static final ConfigEntry<List<String>> AUTHORIZATION_OWNER_PRIVILEGES =
+      new ConfigBuilder("gravitino.authorization.owner.privileges")
+          .doc("The privileges that the owner of a security object has.")
+          .version(ConfigConstants.VERSION_0_7_0)
+          .stringConf()
+          .toSequence()
+          .checkValue(
+              valueList ->
+                  valueList != null && valueList.stream().allMatch(StringUtils::isNotBlank),
+              ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .createWithDefault(Lists.newArrayList());
+
+  public static final ConfigEntry<List<String>> RANGER_POLICY_RESOURCE_DEFINES =
+      new ConfigBuilder("gravitino.authorization.ranger.policy.resource.defines")
+          .doc("The resource defines that are used in Ranger policies.")
+          .version(ConfigConstants.VERSION_0_7_0)
+          .stringConf()
+          .toSequence()
+          .checkValue(
+              valueList ->
+                  valueList != null && valueList.stream().allMatch(StringUtils::isNotBlank),
+              ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .createWithDefault(Lists.newArrayList());
+
+  public static final String PRIVILEGE_MAPPING_PREFIX =
+      "gravitino.authorization.privilege.mapping.";
+
+  public static AuthorizationConfig loadConfig(String provideName) {
+    AuthorizationConfig authorizationConfig = new AuthorizationConfig();
+    boolean testEnv = System.getenv("GRAVITINO_TEST") != null;
+    String propertyFilePath =
+        String.format("authorization-defs/authorization-%s.properties", provideName);
+
+    // Only load from resources in test environment
+    if (testEnv) {
+      URL resourceUrl = AuthorizationConfig.class.getResource(File.separator + propertyFilePath);
+      try {
+        if (resourceUrl != null) {
+          Properties properties =
+              authorizationConfig.loadPropertiesFromFile(new File(resourceUrl.getPath()));
+          authorizationConfig.loadFromProperties(properties);
+          return authorizationConfig;
+        }
+      } catch (IOException e) {
+        throw new IllegalArgumentException(
+            "Failed to load authorization config from resource " + resourceUrl, e);
+      }
+    }
+
+    // Load from configuration directory in production environment
+    String confPath =
+        String.join(
+            File.separator,
+            System.getenv("GRAVITINO_HOME"),
+            "authorizations",
+            "ranger",
+            "conf",
+            propertyFilePath);
+    try {
+      Properties properties = authorizationConfig.loadPropertiesFromFile(new File(confPath));
+      authorizationConfig.loadFromProperties(properties);
+    } catch (IOException e) {
+      throw new IllegalArgumentException(
+          "Failed to load authorization config from file " + confPath, e);
+    }
```

Comment:
Can we merge these two parts and share the confPath?

Reply:
OK